datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchHeader.tsx

```diff
@@ -56,70 +53,63 @@ export default function EmbeddedListSearchHeader({
-                        suggestions={[]}
-                        style={{
-                            maxWidth: 220,
-                            padding: 0,
-                        }}
-                        inputStyle={{
-                            height: 32,
-                            fontSize: 12,
-                        }}
-                        onSearch={onSearch}
-                        onQueryChange={onQueryChange}
-                        entityRegistry={entityRegistry}
-                    />
-                    {/* TODO: in the future, when we add more menu items, we'll show this always */}
-                    {showSelectMode ? (
-                        <>
-                            <Button
-                                style={{
-                                    marginLeft: '5px',
-                                }}
-                                onClick={() => setShowSelectMode(false)}
-                                type="text"
-                            >
-                                <CloseCircleOutlined /> Cancel
-                            </Button>
-                            {/* <SearchMenuContainer>
-                                <SearchActionMenu checkedSearchResults={checkedSearchResults} />
-                            </SearchMenuContainer> */}
-                        </>
-                    ) : (
+        <>
+            <TabToolbar>
+                <HeaderContainer>
+                    <Button type="text" onClick={onToggleFilters}>
+                        <FilterOutlined />
+                        <Typography.Text>Filters</Typography.Text>
+                    </Button>
+                    <SearchAndDownloadContainer>
+                        <SearchBar
+                            initialQuery=""
+                            placeholderText={placeholderText || 'Search entities...'}
+                            suggestions={[]}
+                            style={{
+                                maxWidth: 220,
+                                padding: 0,
+                            }}
+                            inputStyle={{
+                                height: 32,
+                                fontSize: 12,
+                            }}
+                            onSearch={onSearch}
+                            onQueryChange={onSearch}
+                            entityRegistry={entityRegistry}
+                        />
                         <SearchMenuContainer>
                             <SearchExtendedMenu
                                 callSearchOnVariables={callSearchOnVariables}
                                 entityFilters={entityFilters}
                                 filters={filters}
                                 query={query}
-                                setShowSelectMode={setShowSelectMode}
+                                // setShowSelectMode={setIsSelectMode}
```

Comment:
nit ^

Reply:
Oh thanks!

Reply:
Actually this is intentional. We are disabling the select mode (it shouldn't be enabled yet from this PR since no actions are supported)